task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -84,7 +84,7 @@ spec:
         name: workdir
   steps:
     - name: skip-ta
-      image: registry.access.redhat.com/ubi9/ubi-minimal:9.3-1612@sha256:119ac25920c8bb50c8b5fd75dcbca369bf7d1f702b82f3d39663307890f0bf26
```

Comment:
@zregvart `hack/generate-ta-tasks.sh` tries to rollback this change to old image, where is the source for this image URL? it's not in prefetch-dependencies

Reply:
https://github.com/konflux-ci/build-definitions/blob/86571c18bce13d740d091b449c48f90006278ada/task/prefetch-dependencies-oci-ta/0.1/recipe.yaml#L10  foudnd it, but it's out of renovate radar, it cannot update such file

Reply:
is the expected workfow that reviewer will update it manually in order to merge dependabot PR?